README.md

```diff
@@ -12,4 +12,14 @@ This repository includes a sample configuration file to run these on tests on Br
  - run `npm install`
 
 # Run Tests
-- run `npm run wdio`
\ No newline at end of file
+- run `npm run wdio`
+
+# Browserstack Test Observability
```

Comment:
Can we add a small code snippet that show-cases TO integration.
You can add the services part and explain it in a oneLiner.